### PR TITLE
Set Up and Answer Questions

### PR DESCRIPTION
**Pull request by mharley12345**

### CODE REVIEW

File: components/Tabs/index.js [hunk 1]
```diff
@@ -7,3 +7,31 @@
...
+axios.get('https://lambda-times-backend.herokuapp.com/topics')
+.then(response=>{
+     console.log("Data Received:",response)
+          let data = response.data.topics
+        data.forEach(item=>{
+          tabCreator(item)})
+        
+  .then(error=> {
+            console.log('Api down for', error)
+          })
```
Comment (amlane) [inline components/Tabs/index.js]: You'll get much more practice with axios calls. Remember that .then and .catch are the homies. You'll never catch them not hanging out together. Modify your 2nd .then to the .catch and move it outside of your .then call like this:

```axios.get('https://lambda-times-backend.herokuapp.com/topics')
.then(response=>{
     console.log("Data Received:",response)
          let data = response.data.topics
        data.forEach(item=>{
          tabCreator(item)})
         
        
}).catch(error => {
  console.log('Api down for', error)
})

File: components/Carousel/Carousel.js [hunk 2]
```diff
@@ -17,3 +17,51 @@
...
+  leftButton.addEventListener('click',()=>{
+     comp.style.display="block"
+     turnTable.style.display='block'
+     mtn.style.display="none"
+     trees.style.display="none"
+  })
+               
+  return carouselComponent,leftButton,rightButton,turnTable,comp,trees,mtn
+  }
+  carouselComponent()
```
Comment (amlane) [inline components/Carousel/Carousel.js:65]: Looks like you started to attempt stretch! Great job. I don't know very many students that would even attempt this stretch challenge!!

Comment (mharley12345) [reply components/Carousel/Carousel.js:65]: Thank You

**Review by mharley12345 — COMMENTED** [review]
Fixed axios call error. Thanks Amanda

**Pull request merged**